turbopack/crates/turbopack-node/src/evaluate.rs

```diff
@@ -243,18 +251,21 @@ pub async fn get_evaluate_pool(
     debug: bool,
     env_var_tracking: EnvVarTracking,
 ) -> Result<Vc<NodeJsPool>> {
-    let EmittedEvaluatePoolAssets {
-        bootstrap,
-        output_root,
-        entrypoint,
-    } = &*emit_evaluate_pool_assets_with_effects_operation(
+    let operation = emit_evaluate_pool_assets_with_effects_operation(
         module_asset,
         asset_context,
         chunking_context,
         runtime_entries,
-    )
-    .read_strongly_consistent()
-    .await?;
+    );
+    let EmittedEvaluatePoolAssetsWithEffects { assets, effects } =
+        &*operation.read_strongly_consistent().await?;
+    effects.apply().await?;
```

Comment:
The effects need to be applied in `get_evaluate_pool` and not in a cached inner function. Otherwise the cached function is not reexecuted, which leads to missing files.